src/converter/one/util/Frames.java

```diff
@@ -0,0 +1,11 @@
+package one.util;
+
+public class Frames {
+    public static final byte FRAME_INTERPRETED = 0;
+    public static final byte FRAME_JIT_COMPILED = 1;
+    public static final byte FRAME_INLINED = 2;
+    public static final byte FRAME_NATIVE = 3;
+    public static final byte FRAME_CPP = 4;
+    public static final byte FRAME_KERNEL = 5;
+    public static final byte FRAME_C1_COMPILED = 6;
```

Comment:
These constants are currently available in class Frame:
https://github.com/async-profiler/async-profiler/blob/master/src/converter/one/convert/Frame.java#L11-L17

Reply:
done